console/src/main/scripts/plugins/metrics/html/url-list.html

```diff
@@ -66,7 +66,7 @@ <h2 ng-show="vm.resourceList.length > 0" class="text-center hk-fade-in">Current
           </div>
           <div class="col-sm-3 hk-summary-item">
             <a href="/hawkular-ui/url/availability/{{res.id}}">
-              <span class="hk-data" ng-show="res.lastDowntime > 0" am-time-ago="res.lastDowntime"></span>
+              <span class="hk-data" ng-show="res.lastDowntime > 0" am-time-ago="res.lastDowntime" tooltip-trigger tooltip-placement="top" tooltip-popup-delay="1500" tooltip="{{ res.lastDowntime | date:'d MMM yyyy, HH:mm:ss' }}"></span>
```

Comment:
Even Gabriel Agreed that tooltip of 1500ms was too long. Let's move this down to 1000ms.